src/panels/lovelace/cards/hui-picture-glance-card.js

```diff
@@ -134,54 +136,56 @@ class HuiPictureGlanceCard extends NavigateMixin(LocalizeMixin(EventsMixin(Polym
       (config.state_image && !config.entity)) {
       throw new Error('Invalid card configuration');
     }
-
-    this._config = config;
+    this.setProperties({
+      _configEntities: processConfigEntities(config.entities),
+      _config: config,
+    });
   }
 
-  _computeEntitiesDialog(hass, config, entitiesService) {
-    if (config.force_dialog) {
-      return config.entities;
+  _computeEntitiesDialog(hass, entitiesService) {
+    if (this._config.force_dialog) {
```

Comment:
Anything that is computed should not depend on values from `this`. Instead `config` should be passed in too? It makes me wonder if using computed properties is really useful here, as all variables are so interconnected. Might be better to just do it as a response to `hass` changing.

Reply:
Even better: we only have to set them on config change